### DISCUSSION TITLE
Add copy code button to code blocks

### DISCUSSION THREAD

**Discussion by kirasok** [srid/emanote#436] [Ideas]
Add button to copy code from code block

**Comment by srid** [comment]
Sure. PR welcome. This should involve just the HTML (`.tpl`) changes.

**Comment by shivaraj-bh** [comment]
An on-click animation would be a plus, some ideas (from [here](https://community.flake.parts/haskell-flake/nixpkgs-haskell#haskell-shell)):
 
### on-hover
<img width="745" alt="Screenshot 2023-12-06 at 6 04 56 PM" src="https://github.com/srid/emanote/assets/23645788/54e71e96-9f52-45df-8ee3-4b694dda5d65">

### on-click
<img width="758" alt="Screenshot 2023-12-06 at 6 05 59 PM" src="https://github.com/srid/emanote/assets/23645788/7cbae90f-99a5-4d46-9799-f7e5a04b750f">

**Comment by srid** [comment]
https://github.com/srid/emanote/issues/464

**Comment by srid** [comment]
Done in https://github.com/srid/emanote/pull/622
